lib/simp/beaker_helpers.rb

```diff
@@ -2,7 +2,7 @@ module Simp; end
 
 module Simp::BeakerHelpers
   require 'simp/beaker_helpers/version'
-  DEFAULT_PUPPET_AGENT_VERSION = '1.8.3'
+  DEFAULT_PUPPET_AGENT_VERSION = '1.10.2'
```

Comment:
@jeefberkey Yeah, it is. We need to test both old and new so I guess it doesn't really matter which it is set to we just have to remember to run the old ones as well.

@heliocentric Does it make sense to set it to the oldest supported by default?